instaclustr/resource_cluster.go

```diff
@@ -726,9 +727,13 @@ func getBundlesFromCluster(cluster *Cluster) ([]map[string]interface{}, error) {
 
 	bundles := make([]map[string]interface{}, 0)
 	bundles = append(bundles, baseBundle)
-	if cluster.AddonBundles != nil {
-		for _, addonBundle := range cluster.AddonBundles{
-			bundles = append(bundles, addonBundle)	
+	addonBundles := cluster.AddonBundles
+	sort.Slice(addonBundles, func(i, j int) bool {return addonBundles[i]["bundle"].(string) > addonBundles[j]["bundle"].(string)})
+	if addonBundles != nil {
+		for _, addonBundle := range addonBundles {
+			if len(addonBundle) != 0 {
+				bundles = append(bundles, addonBundle)
+			}
```

Comment:
**[Desired, Maintainability]** Would be good to invert the conditional here, would make this more logical and maintainable:
```go
if addonBundles != nil {
    return nil, nil
}
for _, addonBundle := range addonBundles {
    if len(addonBundle) != 0 {
        bundles = append(bundles, addonBundle)
    }
}
```